_dashboards/management/advanced-settings.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Advanced settings
+parent: Dashboards Management
+nav_order: 40
+---
+
+# Advanced settings
+
+Use the **Advanced settings** page to modify settings that govern OpenSearch Dashboards behavior. For example, you can use the page to enable or disable the dark mode, change the default language, change the default number of results displayed on a dashboard, or change the default time zone. See the following sections to learn more about the specific settings.
+
+To modify settings in Advanced Settings, follow these steps:
+
+1. Open the OpenSearch Dashboards main menu, then select **Dashboards Management** > **Advanced settings**.
+2. Search for the setting you want to modify.
+3. Enter a new value for the setting. 
+4. Select **Save changes**.
+
+## Required permissions
+
+<To access Advanced settings in OpenSearch Dashboards, you must have the **Advanced settings** privilege. You can add this privilege to a role by opening the menu, selecting **Dashboards Management**, and then selecting **Roles**.><This is example text and needs to be tailored for Dashboards.>
+
+## General settings
+
+The following table describes the **General** settings:
+
+Setting | Description
+:--- | :---
```

Comment:
This is fine for a first version of the doc page, but I don't think it makes sense to duplicate the info and text on the app page itself and here. In the long term, I think we'd be better off updating the text in the app to be more complete, consistent, and self-explanatory, and keep this doc page more high level without these tables. Users should be able to understand the implications of a setting change from the Advanced Settings page itself, without needing to consult the docs.

Reply:
+1 This can easily go out of sync unless its autogenerated from the source code. Until we can do that, id keep most of these settings out of the doc site since they are pretty self explanatory. We can describe a few settings if necessary, but adding all of them makes it seem like a definitive list.